_tests/integration/log_format_test.go

```diff
@@ -44,37 +44,24 @@ func TestConsoleLogCanBeRestoredFromJSONLog(t *testing.T) {
 }
 
 func createConsoleLog(t *testing.T, workflow string) (string, error) {
-	{
-		cmd := exec.Command(binPath(), "setup")
-		outBytes, err := cmd.CombinedOutput()
-		require.NoError(t, err, string(outBytes))
-	}
-
-	{
-		cmd := exec.Command(binPath(), ":analytics", "off")
```

Comment:
I'm not sure why it started failing now and not earlier, but this is no longer correct. The analytics plugin has been deprecated and removed, I changed the test to disable analytics using the new env var